src/angularWhitespaceRule.ts

```diff
@@ -65,9 +76,42 @@ class InterpolationWhitespaceVisitor extends BasicTemplateAstVisitor implements
   }
 }
 
+class SemicolonTempalteVisitor extends BasicTemplateAstVisitor implements ConfigurableVisitor {
+
+  visitDirectiveProperty(prop: ast.BoundDirectivePropertyAst, context: BasicTemplateAstVisitor): any {
+
+    if (prop.sourceSpan) {
+      const directive = (<any>prop.sourceSpan).toString();
+      let expr = directive.split("=")[1].trim();
+      expr = expr.substring(1,expr.length-1).trim();
+
+      // Note that will not be reliable for different interpolation symbols
+      let error = null;
+
+      if (SemicolonNoWhitespaceRe.test(expr)) {
+        error = 'Missing whitespace after semicolon; expecting \'; expr\'';
+        const internalStart = expr.search(SemicolonNoWhitespaceRe)+1;
```

Comment:
what about adding 
```json

    "whitespace": [
      true,
      "check-branch",
      "check-decl",
      "check-module",
      "check-operator",
      "check-preblock",
      "check-separator",
      "check-type"
    ],
```

to tslint.json and not have a necessity to deal with this in CR? :)

btw. tslint is outdated and some rules are already gone. If you want, I can prepare some PR with proposal.